todoComments/todoComments.md

```diff
@@ -0,0 +1,185 @@
+| 84 | This should be moved to the error case below once issues with this process are under control. If all is well it could go inside the case where an unknown error occurred. | src/scripts/installOED.sh | Unsure | [Github issue #584](https://github.com/OpenEnergyDashboard/OED/issues/584) | Could possibly add this todo into the linked issue |
+| 85 | Verify that user see the message returned, see https://express-rate-limit.mintlify.app/reference/configuration#message | src/server/app.js | Issue Is Created | [Github issue #1206](https://github.com/OpenEnergyDashboard/OED/issues/1206) | Directly related to issue 1206 |
+| 86 | This was causing tests to fail for 3D rejection. This limit seems to be okay but we should find a better solution than upping values just for tests. | src/server/app.js | Needs Issue Created | - | Could not find any related issues. Underlying issues seems to be with the automatic test |
+| 87 | Potentially modify the Morgan logger to use the log API, thus unifying all our logging. | src/server/app.js | Needs Issue Created | - | No related issues |
+| 88 | Check if valid env variables are actually loaded despite the lack of a file, only log if they are not | src/server/config.js | Needs Issue Created | - | TODO is self-explanatory. No implementation in code yet. Needs an issue created. |
+| 89 | These likely need to be updated. | src/server/data/automatedTestingData.js | Unsure | - | References a series of commands to remove all the readings and meters associated withthe test data. This is valuable if you want to run the process again and the<br>meters and readings already exist. No related issues found. |
+| 90 | When this is converted to key/value pairs, the area should be set to the comparable meter(s) and campus is the sum of all 121,000. | src/server/data/websiteData.js | Unsure | - | - |
+| 91 | TODO | src/server/migrations/0.7.0-0.8.0/sql/readings/replace_compressed_readings_2.sql | Related To An Issue | - | TODO states that it is reliant on "having sites pick a site-level reading frequency" which is not done (yet?). I could not find an associated issue for that so not sure what to do with this for now. |
+| 92 | Someday consider averaging days if too many. | src/server/migrations/0.8.0-1.0.0/sql/readings/create_reading_views.sql | Unsure | - | This is a multifaceted issue that requires more guidance. How many days is too many days? How do we want to go about averaging this data? |
+| 93 | It is important that a redoCik is done after this to recreate cik. MUST ADD THIS TO THE 1.1 MIGRATION. | src/server/migrations/1.0.0-2.0.0/sql/cik/alter_cik_table.sql | Unsure | - | I'm assuming that this is wanting for the cik table to be rebuilt or repopulated once the table is dropped? This seems like it will be implemented in the next migraiton.  |
```

Comment:
The note is close. This was added to remind us that this migration needs a special step that must be added. Note that it is now 2.0 (not 1.1) but that is not too important.

Reply:
Should the comment be changed to say 2.0 or is it no longer necessary and can be removed?

Reply:
I'm not too worried about the 1.1 vs 2.0. I think the note can be removed.